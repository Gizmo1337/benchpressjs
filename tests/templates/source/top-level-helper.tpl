<div id="widget-global-footer"></div>
  <script type='text/javascript'>
    var Widget = {"wordpress_home":"{function.wordpressHome}"};
    requirejs(['header']);
  </script>
  <script src="{function.stylesheetLocation}/public/js/header-widget.js"></script>
</div>