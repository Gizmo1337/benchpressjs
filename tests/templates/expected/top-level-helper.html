<div id="widget-global-footer"></div>
  <script type='text/javascript'>
    var Widget = {"wordpress_home":"wordpress-home"};
    requirejs(['header']);
  </script>
  <script src="stylesheet-location/public/js/header-widget.js"></script>
</div>